README.md

```diff
@@ -250,7 +278,23 @@ will take precedence.
 
 Hyrax is available under [the Apache 2.0 license](LICENSE.md).
 
-# Contributing
+# Quick Start for Hyrax development and contributing code back
```

Comment:
@elrayle :speech_balloon: 
> Will the dev guide move to projecthydra.github.io?

That works for me. For now, moving current CC and Sufia wiki docs to Hyrax is my priority in the coming weeks.